sdk/python/feast/repo_config.py

```diff
@@ -347,14 +342,11 @@ def _validate_online_store_config(cls, values):
             online_config_class = get_online_config_from_type(online_store_type)
             online_config_class(**values["online_store"])
         except ValidationError as e:
-            raise ValidationError(
-                [ErrorWrapper(e, loc="online_store")],
-                model=RepoConfig,
-            )
+            raise e
         return values
 
-    @root_validator(pre=True)
-    def _validate_offline_store_config(cls, values):
+    @model_validator(mode="before")
```

Comment:
turns out also need to add classmethod for `model_validator` too
https://docs.pydantic.dev/latest/concepts/validators/#model-validators